core/src/main/java/org/radargun/stages/test/legacy/LegacyStressor.java

```diff
@@ -94,8 +96,13 @@ private void runInternal() {
             Operation operation = operationSelector.next(random);
             try {
                logic.run(operation);
+               if (delayBetweenRequests > 0)
```

Comment:
Well, if you think that the results will be of any use to you, go for it. I just won't explain anyone why the results look the way they do :)